index.js

```diff
@@ -73,15 +73,21 @@ function makeHash (markdownString, opts) {
   return hasha(markdownString + optsString)
 }
 
-function createProcessor (before) {
+function createProcessor (before, ignoreMissing) {
```

Comment:
Hmm it seems like one of my comments got lost.  I think this `ignoreMissing` should be renamed for clarity. What do you think about allowing a `highlight` option to be passed instead?

```js
{
  highlight: {
    ignoreMissing: true
  }
}
```

Reply:
@zeke 4ef1dece829864a730700319494b9ee2df742131 introduces the `highlight` object which can accept all options from [rehype-highlight](https://github.com/rehypejs/rehype-highlight#options) module